jdaviz/configs/default/plugins/plot_options/plot_options.vue

```diff
@@ -366,9 +360,9 @@
       <v-row>
         <v-text-field
             ref="stretch_hist_nbins"
-            type="number"
             label="Number of Bins"
             v-model.number="stretch_hist_nbins"
+            type="number"

```

Comment:
What is this point of moving this 3 lines down? Just want to make sure this is intentional.

Reply:
just to try to be consistent with it listed below `v-model.number` since I suspect having it not directly associated may have been what resulted in the duplicated lines in other instances also fixed here.